<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammersCalculator</class>
 <widget class="QMainWindow" name="ProgrammersCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgrammersCalculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="radixListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>85</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Liberation Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="stackListWidget">
      <property name="font">
       <font>
        <family>Liberation Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
